doc/api/prev_api_changes/api_changes_0.91.0.rst

```diff
@@ -31,7 +31,7 @@ Changes for 0.91.0
   files. In the future the class might actually parse the font to
   allow e.g.,  subsetting.
 
-* :mod:`matplotlib.ft2font` now supports ``FT_Attach_File``. In
```

Comment:
This is a bug in the docs that we do not have an entry for `matplotlib.ft2font` (which to be fair also lacks a module docstring).